app/workflows/lists/conversion/definition.yml

```diff
@@ -1,477 +1,7 @@
-        actions:
-          - title: Email final checklist to external stakeholders
-          - title:
-              Receive email of responses to final checklist from external
-              stakeholders
-          - title:
-              Save final checklist email response from external stakeholders in
-              SharePoint
-
-      - title: Confirm all conditions met
-        actions:
-          - title: CTA agreed and signed (commercial transfer agreement)
-          - title: Leases agreed and signed
-          - title: RPA (risk protection arrangements) confirmed
-          - title: New bank account sent via online form
-          - title:
-              Final re-opening actions (checklist sent to academy to complete)
-          - title: Confirm have signed documents from the Trust
-          - title:
-              Email the operational academy team to confrim that all conditions
-              are met and the school is ready for opening
-
-      - title: Sign & seal all model documents
-        actions:
-          - title:
-              Extract pages required for signature by the secretary of state
-              (these are delegated to the Team Lead)
-          - title: Send extracted pages to the Team Lead for signing
-          - title: Insert signed pages back into required documents
-
-      - title: Notify school / trust
-        actions:
-          - title:
-              Confirm the school can convert to the trust on the agreed date
-              (1st of the folloiwng month)
-          - title: Confirm the new URN number
-          - title:
-              Send signed and sealed documents back to Trust and solicitor team
-          - title: Send blank grant expenditure certificate to complete
-
-  - title: Project close
-    tasks:
-      - title: Redact, save and share documents
-        actions:
-          - title: Redact required documents and save in SharePoint
-          - title:
-              Send redacted documents to "Funding Agreements converter" mail box
-          - title:
-              Ensure all documents and key information are saved in SharePoint
-
-      - title: ESFA final checks and handover
-        actions:
-          - title: Ensure all ESFA fields are up to date in KIM
-          - title: Add any ESFA handover comments in KIM
-          - title: Ensure single worksheet is up to date
-
-      - title: Handback with regional delivery officer
-        actions:
-          - title: Complete handback proforma
-          - title: Save handback proforma in SharePoint
-          - title: Send completed handback proforma to regional delivery officer
-
-      - title: Complete processing of conversion grant
-        actions:
-          - title: Receive completed grant certificate from trust
-          - title: Save completed grant certificate to SharePoint
-          - title:
-              Update the "support grant certificate assurance report
-              spreadsheet"
+  - kickoff
```

Comment:
Could this file be called `tasklist` to match up with the language everywhere else?